modules/transactions.js

```diff
@@ -102,6 +102,8 @@ __private.list = function(filter, cb) {
 	const params = {};
 	const where = [];
 	const allowedFieldsMap = {
+		senderIdOrRecipientId:
+			'"t_senderId" IN (${senderIdOrRecipientId}) OR "t_recipientId" IN (${senderIdOrRecipientId})',
```

Comment:
@Tschakki That needs to be changed, now as you are doing checking for only one variable then you should do with the direct assignment. 

```
"t_senderId" = (${senderIdOrRecipientId}) OR "t_recipientId" = (${senderIdOrRecipientId})
```